App/Reducers/LoginReducer.js

```diff
@@ -9,20 +9,24 @@ export const INITIAL_STATE = Immutable({
 })
 
 // login attempts
-const attempt = (state, action) =>
-  state.merge({ attempting: true })
+const attempt = (state, action) => {
+  return { ...state, ...{ attempting: true } }
```

Comment:
Cool. I've fixed the syntax. 

@skellock I have no clue why the new `state` object doesn't have the `merge` function on it, but when i added in `redux-saga` the previous `state.merge(x)` was throwing an error: `state.merge is not a function`.